searx/engines/onesearch.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+
+"""OneSearch (Yahoo & Verizon)
+
+- https://www.onesearch.com
+
+OneSearch is literally just Bing results flanked by ads that don’t track you
+despite being from a company that makes money tracking you [1].
+
+According to the OneSearch privacy policy, search results will only be
+personalized based on location, which it will collect from IP addresses.
+OneSearch says that it will separate IP addresses from users and their search
+results [2].
+
+[1] https://lifehacker.com/is-yahoos-new-onesearch-engine-good-for-privacy-1841042875
+[2] https://www.theverge.com/2020/1/14/21065640/verizon-onesearch-privacy-tracking-yahoo-breach-hack
```

Comment:
compare with https://github.com/searxng/searxng/pull/513/files#r751782506

Reply:
I've added a wikidata entry: https://www.wikidata.org/wiki/Q109682354 so `update_engine_descriptions.py` should the wikidata description.

Reply:
> I've added a wikidata entry

Very cool .. 

https://github.com/searxng/searxng/blob/c4a8c77026319185f516567f9b288157c2f87e20/searx/engines/onesearch.py#L28-L30

and I can confirm update_engine_descriptions.py works:

https://github.com/searxng/searxng/blob/356cff0692e0c59d263c17964dfb3a45ee3b2305/searx/data/engine_descriptions.json#L1053-L1056